### DISCUSSION TITLE
[Question] How to not clear the tree-select dropdown

### DISCUSSION THREAD

**Discussion by NareshMurthy** [ant-design/ant-design#43965] [Q&A]
### What problem does this feature solve?

I am using the tree-select with Multiple selection
After searching in the input field and selecting an option, Is it possible to keep the dropdown view as is (without reset), but only clear the text that was searched in the input

I tried `autoClearSearchValue={false}` and `searchValue={inputSearchText}` and trying to clear the only the inputSearchText

Thanks!

**Comment by MadCcc** [comment]
Not supported, search value is controlled for filtering.
But you can customize dropdown with `dropdownRender`.

**Comment by NareshMurthy** [comment]
Hi @MadCcc 
Is it possible to do with it `dropdownRender` ?

**Comment by MadCcc** [comment]
> Hi @MadCcc Is it possible to do with it `dropdownRender` ?

I mean you could manage your own `searchValue` that will not be cleared, and filter `treeData` on your own.
